hazelcast/src/main/java/com/hazelcast/map/impl/tx/TxnUnlockBackupOperation.java

```diff
@@ -31,18 +31,20 @@
 public class TxnUnlockBackupOperation extends KeyBasedMapOperation implements BackupOperation {
 
     private String ownerUuid;
+    private long lockThreadId;
 
     public TxnUnlockBackupOperation() {
     }
 
-    public TxnUnlockBackupOperation(String name, Data dataKey, String ownerUuid) {
-        super(name, dataKey, -1, -1);
+    public TxnUnlockBackupOperation(String name, Data dataKey, String ownerUuid, long lockThreadId) {
+        super(name, dataKey);
         this.ownerUuid = ownerUuid;
+        this.lockThreadId = lockThreadId;
```

Comment:
How come we don't reuse `KeyBasedMapOperation#threadId` since it's already there?